examples/8a_benchmark_write_parallel.cpp

```diff
@@ -460,15 +465,19 @@ int parseArgs( int argc, char *argv[], TestInput& input )
     if( argc >= 5 )
         input.m_Steps = atoi( argv[4] );
 
-
     if (argc >= 6)
       input.m_Dim = atoi( argv[5] );
 
-    //if ( (dataDim > 3) || (dataDim < 0) )  {
-    //return -1;
-    //}
+    if (argc >= 7) {
```

Comment:
Please don't forget to document the new option in `docs/source/usage/benchmarks.rst` so people know how to use it :)

Reply:
This option is making sense for bp users. The benchmark right now has fast track options for H5 users.  Will need to add another section later.